scripts/create-s3-bucket.bash

```diff
@@ -11,5 +11,5 @@ IFS=$'\n\t'
 docker run \
 -v /etc/ssl/certs:/etc/ssl/certs:ro \
 --network host \
---env MC_HOST_local="https://${S3_ACCESS_KEY}:${S3_SECRET_KEY}@${S3_ENDPOINT}" \
-minio/mc:RELEASE.2023-06-19T19-31-19Z mb --ignore-existing local/"$1"
+--env MC_HOST_local="https://${S3_ACCESS_KEY}:${S3_SECRET_KEY}@${STORAGE_DOMAIN}" \
```

Comment:
Why change to `STORAGE_DOMAIN`? The script is called `create-s3-bucket.bash` and `S3_ENDPOINT` seems to be a more appropriate candidate in this context

Reply:
`STORAGE_DOMAIN` and `S3_ENDPOINT` encode the same domain, but afaik `S3_ENDPOINT` also contains the prefix `https://`. This is just a quick change of env-vars that are anyway available to make the script work. The script didnt work anymore with `S3_ENDPOINT`

Reply:
This is true